bitnami/etcd/templates/_helpers.tpl

```diff
@@ -79,12 +79,8 @@ Return the proper etcd client protocol
 Return the proper etcd data dir
 */}}
 {{- define "etcd.dataDir" -}}
-{{- if .Values.persistence.enabled -}}
-{{- print "/bitnami/etcd/data" -}}
-{{- else -}}
 {{- print "/opt/bitnami/etcd/data" -}}
 {{- end -}}
-{{- end -}}
```

Comment:
Did you test this change? What happens when you disable the persistence?

Reply:
Worked fine when I quickly deployed without persistence for a smoke test.